tensorrt_llm/sampling_params.py

```diff
@@ -224,13 +225,16 @@ class SamplingParams:
     beam_width_array: Optional[List[int]] = None
 
     # Keep the below fields in sync with tllme.OutputConfig
-    return_log_probs: bool = False
+    logprobs: Optional[int] = None
+    prompt_logprobs: Optional[int] = None
     return_context_logits: bool = False
     return_generation_logits: bool = False
     exclude_input_from_output: bool = True
     return_encoder_output: bool = False
     return_perf_metrics: bool = False
     additional_model_outputs: Optional[List[AdditionalModelOutput]] = None
+    _context_logits_auto_enabled: bool = False
```

Comment:
Can you add some comments here to record the background for the WAR? Since this is TRT-only and SamplingParams is shared between both TRT and PyT, we may deprecate it in the future.

Reply:
updated